casper/src/main/scala/coop/rchain/casper/api/RuntimeAPI.scala

```diff
@@ -0,0 +1,71 @@
+package coop.rchain.casper.api
+
+import cats.syntax.all._
+import cats.effect.Concurrent
+import com.google.protobuf.ByteString
+import coop.rchain.models.syntax._
+import coop.rchain.casper.syntax._
+import coop.rchain.blockstorage.BlockStore
+import coop.rchain.casper.MultiParentCasper
+import coop.rchain.casper.engine.EngineCell
+import coop.rchain.casper.engine.EngineCell.EngineCell
+import coop.rchain.casper.util.rholang.RuntimeManager
+import coop.rchain.models.{BindPattern, Par}
+import coop.rchain.models.rholang.sorter.Sortable.parSortable
+
+object RuntimeAPI {
+
+  def getFromRuntime[F[_]: Concurrent: EngineCell: BlockStore, A, O](
```

Comment:
Why pull bock store and engine cell here? And not just RuntimeManager + input stateHash instead of blockHash

Reply:
Then I don't actually need to implement any codes. Just use RuntimeManager API Then. @tgrospic agree?

Reply:
We can ask Stas to do the next.

Reply:
@zsluedem We can use `getData` on RuntimeManager (right now we don't need to get continuation). This can be very similar method added to BlockAPI like exploratory deploy but instead of executing Rholang we just call getData on RM.

Reply:
@tgrospic Not sure what you mean. Is my current pr needed?

Reply:
Yes, I'm saying this new method can be on BlockAPI almost the same as exploratory deploy but instead of accepting Rholang term it should accept Par.